src/api/lib/authenticator.rb

```diff
@@ -225,6 +225,7 @@ def check_extracted_user
     end
 
     User.session = @http_user
+    Feature.use_beta_features(@http_user.in_beta?)
```

Comment:
I also do not like this here too much as the feature selection is now leaking into app code and spec code.

Reply:
yeah, because one global state (active features) is intermixed with another global state (active user) - and feature gem wasn't made for that. It's actually 2 different feature sets that the user should pick. But that would mean duplicating the features API from within the user model.

Would you prefer that?

Reply:
I prefer what we have. Extending the gem in `lib/`.

Reply:
We are coupling now the authenticator lib with the beta feature.

Reply:
> We are coupling now the authenticator lib with the beta feature.

What does this have to do with the authenticator?

Reply:
> What does this have to do with the authenticator?

This PR wiill couple it with the features. I'm just saying what it is doing, not that is right.